source/en/routedem.rst

```diff
@@ -28,6 +28,10 @@ The freshwater models in InVEST are routed across a digital elevation model (DEM
 
 * Create streams based on flow accumulation.
 
+* Determine the Strahler Stream Order of streams.
```

Comment:
I mentioned this over in the InVEST PR too but would it be worth elaborating on what Strahler Stream order is? I don't think it's mentioned at all in the UG. Or is it common knowledge enough for us not to have to have an explainer?

Reply:
Yeah, this is new in the InVEST context (not that we actually use Strahler orders in InVEST), but I think this is something that's pretty standard in hydrology.  It seems to me that if we need to explain Strahler stream orders, then we should probably also explain flow accumulation!  What do you think?

Reply:
Let's punt on it! Feels more like a science communication thing and it'd be great if we could defer to other NatCappers for that.